public/css/style.scss

```
@@ -650,49 +685,44 @@ a.social {
 
 .segment {
 	padding: 60px 0 70px 0;
-	background-color: #fff;
+	background-color: light-dark($segmentBg, $segmentBgDark);
+	color: light-dark($segmentText, $segmentTextDark);
 	.page-header {
 		border-bottom-color: #d4d4d4;
 	}
 
-
 	&#home1 {
 		.col-lg-4 {
-			h4, p {
+			h4,
+			p {

```

This also shortens the resulting `.css` by combining otherwise duplicated selectors.

```suggestion
			:where(h4, p) {
```